.ado/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "timeout": "5s",
+  "aliveStatusCodes": [200, 206, 0, 503]
+}
```

Comment:
What is this file?

Reply:
I think it's used to determine if a link is alive or dead. "timeout" specifies the amount of time to wait, "aliveStatusCodes" are HTTP codes to consider as alive.

Reply:
I found that out here: https://github.com/tcort/markdown-link-check#config-file-format

Reply:
Because this otherwise looks like a very generic config file, I wonder if it's possible to put a comment above or below the JSON specifying this is used specifically for the markdown link tester?

Reply:
Sure, I can add one

Reply:
I guess you can't add comments to JSON files (unless you added a comment as a data entry) so I just renamed the file to markdown-link-check-config.json